core/commonTest/src/stress/set/PersistentOrderedSetTest.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016-2025 JetBrains s.r.o.
+ * Use of this source code is governed by the Apache 2.0 License that can be found in the LICENSE.txt file.
+ */
+
+package tests.stress.set
+
+import kotlinx.collections.immutable.persistentSetOf
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class PersistentOrderedSetTest {
```

Comment:
It would be nice to cover `PersistentHashSet` with tests similar to what you added for the ordered set (and the underlying map).